test/introspection/ERC165Checker.test.js

```diff
@@ -0,0 +1,87 @@
+const ERC165Checker = artifacts.require('ERC165CheckerMock');
+const ERC165NotSupported = artifacts.require('ERC165NotSupported');
+const ERC165GenerallySupported = artifacts.require('ERC165GenerallySupported');
+const ERC165InterfaceSupported = artifacts.require('ERC165InterfaceSupported');
+
+const DUMMY_ID = '0xdeadbeef';
+
+require('chai')
+  .should();
+
+contract('ERC165Checker', function (accounts) {
+  const assertIgnoringCoverage = function (actual, expected) {
```

Comment:
Could you add a little comment where explaining _why_ we might need to skip these assertions (maybe with a link to the relevant comments in this PR)?